code/modules/custom_ka/projectiles.dm

```diff
@@ -8,37 +8,55 @@
 	range = 5
 	var/pressure_decrease = 0.25
 
+	// Two variables below are use only to transfer damage scale and aoe variable between parent Collide proc and other procs
+	var/damage_s
+	var/aoe_s = 1
+
 /obj/item/projectile/kinetic/mech
-	damage = 15
+	damage = 40
+	aoe = 5
 
 /obj/item/projectile/kinetic/mech/burst
-	damage = 10
+	damage = 25
 
-/obj/item/projectile/kinetic/on_impact(var/atom/A,var/aoe_scale = 1, var/damage_scale = 1)
+/obj/item/projectile/kinetic/Collide(var/atom/A)
 	var/turf/target_turf = get_turf(A)
 	if(!target_turf)
 		target_turf = get_turf(src)
 	if(istype(target_turf))
 		var/datum/gas_mixture/environment = target_turf.return_air()
+		if(damage_s)
+			damage = damage_s
+		else
+			damage *= max(1 - (environment.return_pressure()/100)*0.75,0)
+			damage_s = damage
+		if(isliving(A)) //Never do more than 50 damage to a living being per shot.
+			damage = min(damage, 50)
+
+	
+	damage_s = damage
```

Comment:
This _also_ doesn't make sense. What is the point of a multiplier (damage scale) if we just make it equal damage?